torch/csrc/autograd/init.cpp

```diff
@@ -593,6 +594,57 @@ static PyObject* is_autocast_available(
   END_HANDLE_TH_ERRORS
 }
 
+static PyObject* set_cached_tensors_enabled(PyObject* _unused, PyObject* arg) {
```

Comment:
Why did you choose to bind these by hand and not use pybind as in the previous code?
I would also suggest putting these in the main namespace and not autograd: https://github.com/pytorch/pytorch/blob/ca5d13c67287efe78a09dcf9b06f7e4fba898689/torch/csrc/Module.cpp#L1760

Reply:
I think `cached tensor` is mainly used in autocast and autocast is under autogard namespace. So I put `cached tensor` to autograd namespace and bind them by hand. It is OK to me to move them to main namespace.